src/jit/compiler.cpp

```diff
@@ -4359,6 +4359,7 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
         bool doCopyProp      = true;
         bool doAssertionProp = true;
         bool doRangeAnalysis = true;
+        int  iterations      = 1;
```

Comment:
Ok. But specifying zero is bad currently -- same as specifying something very large due to "do" loop.